### PR TITLE
EX-1496 - Style the sicherearbeit.at CTA with background color/image

### PR DESCRIPTION
**Pull request by Bejor**
Bildschirmfoto 2021-09-24 um 13 38 44
Bildschirmfoto 2021-09-24 um 13 38 12

### CODE REVIEW

File: src/components/ex-cta-box/cta-box.scss [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+    .cta__text {
+      @apply font-menu text-default flex-grow font-light text-black;
+    }
+
+    .cta__button {
+      @apply bg-primary inline-block text-white transition-all duration-100 ease-in-out;
+
+      &:hover {
+        @screen sm {
```
Comment (martinkastler) [inline src/components/ex-cta-box/cta-box.scss:106]: There is no mouse-ever effect on mobile. It should work on mobile as well.

Comment (martinkastler) [reply src/components/ex-cta-box/cta-box.scss:106]: This comment is in the wrong place. I refer to the white buttons at the end of the box.

File: src/components/ex-cta-box/cta-box.scss [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    .cta__text {
+      @apply text-default mt-2 font-normal text-center text-white;
+
+      p {
+        @apply mt-0;
+      }
+    }
+
+    .cta__button {
+      @apply text-default text-primary flex items-center text-left uppercase transition-all duration-100 ease-in-out bg-white;
```
Comment (martinkastler) [inline src/components/ex-cta-box/cta-box.scss:149]: Seems in the tablet break point the button label gets left-aligned. Should always be centered:

image

**Review by martinkastler — CHANGES_REQUESTED** [review]

File: src/components/ex-cta-box/cta-box.scss (hunk 1 above)
Comment (Bejor) [reply src/components/ex-cta-box/cta-box.scss]: Activated hover for mobile

File: src/components/ex-cta-box/cta-box.scss (hunk 2 above)
Comment (Bejor) [reply src/components/ex-cta-box/cta-box.scss]: Text is now center

File: src/components/ex-cta-box/cta-box.scss (hunk 1 above)
Comment (martinkastler) [reply src/components/ex-cta-box/cta-box.scss]: Good start. The CTA in red color as black font color on navy background. Should be white background:

image

File: src/components/ex-cta-box/cta-box.scss (hunk 2 above)
Comment (martinkastler) [reply src/components/ex-cta-box/cta-box.scss]: Yep on mobile and desktop. It seems the tablet viewport is still missing the style:

image

File: src/components/ex-cta-box/cta-box.scss (hunk 1 above)
Comment (Bejor) [reply src/components/ex-cta-box/cta-box.scss]: should be fixed

File: src/components/ex-cta-box/cta-box.scss (hunk 2 above)
Comment (Bejor) [reply src/components/ex-cta-box/cta-box.scss]: fixed

**Review by martinkastler — APPROVED** [review]

**Pull request merged**